pkg/registry/serving/composer.go

```diff
@@ -182,11 +211,13 @@ func getModelFormat(pu *seldonv1.PredictiveUnit) string {
 }
 
 // getUserContainerImage get image by different model format.
-func getUserContainerImage(pu *seldonv1.PredictiveUnit) string {
-	format := getModelFormat(pu)
+func getUserContainerImage(format string) string {
```

Comment:
I am not sure if we should set a server with environment variables, maybe we should have an elegant design in the future.

Reply:
I think we should get the format from ormbfile.

Reply:
/cc @simon-cj

Reply:
Because we pass the format params by frontend, user need choose format in UI in Clever, if we remove it, the product need change too.